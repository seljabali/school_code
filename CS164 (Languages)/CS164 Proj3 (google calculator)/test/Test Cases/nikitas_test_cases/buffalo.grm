//Buffalo+ is a valid english sentence!
%%
S -> NP VP;
NP -> NP RC | PN N;
RC -> NP V;
VP -> V NP;
PN -> 'Buffalo';
N -> 'buffalo';
V -> 'buffalo';
// try parsing "Buffalo buffalo Buffalo buffalo buffalo buffalo Buffalo buffalo"!