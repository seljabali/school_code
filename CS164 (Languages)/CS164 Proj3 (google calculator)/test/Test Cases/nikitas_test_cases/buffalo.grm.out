(Grammar
    (Declarations
    )
    (Rules
        (Rule S
            (production [ NP, VP, ]
            )
        )
        (Rule NP
            (production [ NP, RC, ]
            )
            (production [ PN, N, ]
            )
        )
        (Rule RC
            (production [ NP, V, ]
            )
        )
        (Rule VP
            (production [ V, NP, ]
            )
        )
        (Rule PN
            (production [ Buffalo, ]
            )
        )
        (Rule N
            (production [ buffalo, ]
            )
        )
        (Rule V
            (production [ buffalo, ]
            )
        )
    )
    (StartSymbol S)
)
